### PR TITLE
bitmex - comments [ci skip]

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/bitmex.ts [hunk 1]
```diff
@@ -807,57 +809,57 @@ export default class bitmex extends Exchange {
...
-        //             "trdMatchID": "string",
-        //             "execCost": 0,
-        //             "execComm": 0,
-        //             "homeNotional": 0,
-        //             "foreignNotional": 0,
-        //             "transactTime": "2019-03-05T12:47:02.762Z",
-        //             "timestamp": "2019-03-05T12:47:02.762Z"
-        //         }
-        //     ]
+        //    [
```
Comment (ttodua) [inline ts/src/bitmex.ts:812]: to be similar format as under 273th line

**Comment by ttodua** [comment]
@carlosmiei I am done here, no more updates needed.

File: ts/src/bitmex.ts [hunk 2]
```diff
@@ -289,95 +290,96 @@ export default class bitmex extends Exchange {
...
-        //        "makerFee": -0.0001,
-        //        "takerFee": 0.0005,
-        //        "settlementFee": 0,
-        //        "insuranceFee": 0,
-        //        "fundingBaseSymbol": ".LTCBON8H",
-        //        "fundingQuoteSymbol": ".USDTBON8H",
-        //        "fundingPremiumSymbol": ".LTCUSDTPI8H",
-        //        "fundingTimestamp": "2022-01-14T20:00:00.000Z",
+        //        "makerFee": "-0.0001",
+        //        "takerFee": "0.00075",
```
Comment (carlosmiei) [inline ts/src/bitmex.ts:322]: @ttodua I don't thing the API returns stringified numbers 🤔

Comment (ttodua) [reply ts/src/bitmex.ts:322]: hm let me recheck what went wrong on my side.

Comment (ttodua) [reply ts/src/bitmex.ts:322]: dang, I posted stringified comments yeah, my bad.

Comment (ttodua) [reply ts/src/bitmex.ts]: seems there were no more comments needed (as previous stringified comments were mistake) so, only these few things were left

**Comment by carlosmiei** [comment]
LGTM ✅

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**